mappings/net/minecraft/item/ArmorMaterial.mapping

```diff
@@ -1,11 +1,83 @@
 CLASS net/minecraft/class_1741 net/minecraft/item/ArmorMaterial
+	COMMENT Defines the material stats of an {@link net.minecraft.item.ArmorItem} item.
+	COMMENT
+	COMMENT <p>
+	COMMENT To view available vanilla armor materials, visit {@link net.minecraft.item.ArmorMaterials}.
 	METHOD method_24355 getKnockbackResistance ()F
+		COMMENT Returns the knockback resistance value of an {@link net.minecraft.item.ArmorItem} piece using this {@link ArmorMaterial}.
```

Comment:
Use inline return? `{@return aaaa}` becomes `Returns aaaa.`

Reply:
Any preference on whether the inline return should be at the top or bottom line of the doc comment? I kept it on the first line in this case.

Reply:
Inline return always comes on the first line.